src/operation_models/unit_commitment.jl

```diff
@@ -1 +1,19 @@
-abstract type UnitCommitment <: AbstractOperationsModel end
\ No newline at end of file
+struct UnitCommitment <: AbstractOperationsModel end
+
+function UnitCommitment(system::PSY.PowerSystem, transmission::Type{S}; optimizer::Union{Nothing,JuMP.OptimizerFactory}=nothing, kwargs...) where {S <: PM.AbstractPowerFormulation}
```

Comment:
as a user, I find that I want this function (constructor) available in a standard naming convention like `build_UnitCommitment` or something similar so that I can start typing what I want to do and view available options with a tab complete.
@kdheepak suggested that a `build` module might be appropriate

Reply:
I can address the naming for the op_models. I don't think to have a separate module for builds is a good idea